src/main/java/AlexLink/HomeWork/HW1/Task4.java

```diff
@@ -0,0 +1,18 @@
+package AlexLink.HomeWork.HW1;
```

Comment:
Seems that you not correctly understand what you need to do.

I enter:
1234
You print:
4321